### PR TITLE
Add tensorboard support

### PR DESCRIPTION
**Pull request by jenniew**
If user set tensorboard dir, train and validation summary would be save to this directory.
If user only specify model_dir, train and validation summary would be save to model_dir.
If neither tensorboard dir nor model_dir were specified, summary would not be saved.

Related issue: https://github.com/analytics-zoo/orca/issues/56

### CODE REVIEW

File: pyzoo/zoo/orca/learn/tf/estimator.py [hunk 1]
```diff
@@ -47,6 +45,76 @@ def load_latest_checkpoint(self, path):
...
+        'HitRatio'                  |   'HitRate@k' (k is Top-k)
+        'Loss'                      |   'Loss'
+        'MAE'                       |   'MAE'
+        'NDCG'                      |   'NDCG'
+        'TFValidationMethod'        |   '${name + " " + valMethod.toString()}'
+        'Top5Accuracy'              |   'Top5Accuracy'
+        'TreeNNAccuracy'            |   'TreeNNAccuracy()'
+        'MeanAveragePrecision'      |   'MAP@k' (k is Top-k) (BigDL)
+        'MeanAveragePrecision'      |   'PascalMeanAveragePrecision' (Zoo)
+        'StatelessMetric'           |   '${name}'
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf/estimator.py:102]: please file an issue wrt the inconsistency between the names

**Comment by jason-dai** [comment]
Looks good to me.

**Comment by jason-dai** [comment]
LGTM

**Comment by jenniew** [comment]
Jenkins pass: http://10.239.47.210:18888/job/ZOO-PR-Validation/3983/

**Pull request merged**